### DISCUSSION TITLE
python import models include mxnet cause a segmentation error

### DISCUSSION THREAD

**Discussion by beingjoey** [apache/incubator-mxnet#14508] [Q&A]
when I just import some model , I get a segmentation fault as below:
from PySide2.QtWidgets import QMainWindow,QMenu,QAction
from PySide2.QtGui import QImage,QPixmap,QCursor
from PySide2.QtCore import QTimer,QSize,QDateTime
import numpy as np
import mxnet as mx
from skimage import transform

Segmentation fault: 11

Stack trace returned 10 entries:
[bt] (0) /usr/local/lib/python2.7/dist-packages/mxnet/libmxnet.so(+0x3e95ea) [0x7f9a489d45ea]
[bt] (1) /usr/local/lib/python2.7/dist-packages/mxnet/libmxnet.so(+0x34f39a6) [0x7f9a4bade9a6]
[bt] (2) /lib/x86_64-linux-gnu/libc.so.6(+0x354b0) [0x7f9ae2d2f4b0]
[bt] (3) python() [0x49d724]
[bt] (4) python() [0x49d8a5]
[bt] (5) python() [0x49c1d7]
[bt] (6) python(_PyObject_GC_Malloc+0x81) [0x4186c1]
[bt] (7) python(PyDict_New+0x11c) [0x49511c]
[bt] (8) python() [0x4de9f5]
[bt] (9) python(PyEval_EvalFrameEx+0x67f6) [0x4c26d6]

my pyside2 version “5.12.1”
numpy version ‘1.14.0’
mxnet version ‘1.4.0’
skimage version ‘0.13.1’

or just import like this:
import PySide2
import numpy
import mxnet
from skimage import transform
I get the same error.

[a comment by an automated account is not shown]

**Comment by beingjoey** [comment]
Now I covered this problem by removing skimage transform model. But I am still worried about that.

**Comment by vdantu** [comment]
@beingjoey : Could you please elaborate what fixed this issue? Did you just remove the 
```
from skimage import transform
```
statement, or did you remove some model altogether? 

@mxnet-label-bot add [question]

**Comment by beingjoey** [comment]
yes, I just remove 
from skimage import transform

**Comment by beingjoey** [comment]
I reproduced this problem on two different computers, with different version of these models.
I found I just can not import PySide2, numpy, mxnet and skimage.transform together. I still don't know whose bug, but the stack trace always showed libmxnet.so .
Mxnet was the core of my app, so skimage.transform was replaced.
